SECURITY.md

```diff
@@ -0,0 +1,79 @@
+how vulnerabilities should be reported.
+
+
+## Vulnerability Definition
+
+WebGPU introduces a different threat model than is sometimes applied to
+GPU-related software. Unlike typical gaming or high-performance computing
+applications, where the software accessing GPU APIs is proprietary or
+obtained from a trusted developer, WebGPU makes GPU APIs available to
+arbitrary web applications. In the threat model of the web, malicious
+content should not be able to use the GPU APIs to access data or interfaces
+outside the intended scope for interaction with web content.
+
+While the WGPU maintainers have discretion in assigning a severity to
+individual vulnerabilities, it is generally considered a security vulnerability
+in WGPU if Javascript or WebAssembly code running with privileges of ordinary
+web content is able to:
+
+- Access data associated with native applications other than the user agent,
+  or associated with other web origins.
+- Escape the applicable sandbox and run arbitrary code or call arbitrary system
+  APIs on the user agent host.
+- Consume system resources to the point that it is difficult to recover
+  (e.g. by closing the web page).
+
+The WGPU Rust API offers some functionality, both supported and experimental,
+that is not part of the WebGPU standard and is not made available in Javascript
+environments using WGPU. Associated vulnerabilities may be assigned lower
+severity than vulnerabilities that apply to a Javascript WebGPU implementation
+based on WGPU.
+
+
+## Supported Versions
+
+The WGPU project maintains security support for serious vulnerabilities in the
+[most recent major release](https://github.com/gfx-rs/wgpu/releases). Fixes for
+security vulnerabilities found shortly after the initial release of a major
+version may also be provided for the previous major release.
+
+Mozilla provides security support for versions of WGPU used in [current
+versions of Firefox](https://whattrainisitnow.com/).
+
+The version of WGPU that is active can be found in the Firefox repositories:
+
+- [release](https://github.com/mozilla-firefox/firefox/blob/release/gfx/wgpu_bindings/Cargo.toml),
+- [beta](https://github.com/mozilla-firefox/firefox/blob/beta/gfx/wgpu_bindings/Cargo.toml), and
+- [nightly](https://github.com/mozilla-firefox/firefox/blob/main/gfx/wgpu_bindings/Cargo.toml),
+
+We welcome reports of security vulnerabilities in any of these released
+versions or in the latest code on the `trunk` branch.
+
+
+## Reporting a Vulnerability
+
+Although not all vulnerabilities in WGPU will affect Firefox, Mozilla accepts
+all vulnerability reports for WGPU and directs them appropriately. Additionally,
+Mozilla serves as the CVE numbering authority for the WGPU project.
+
+To report a security problem with WGPU, create a bug in Mozilla's Bugzilla
+instance in the [Core ::
+Graphics :: WebGPU](https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&component=Graphics%3A+WebGPU)
+component.
```

Comment:
style: This seems oddly reflowed. 🤔 Did you use Prettier on this?

Reply:
It was how my editor reflowed it, prettier doesn't seem to have an opinion. I put the entire link on its own line since that seems more readable.